src/Tasks/ManifestUtil/SecurityUtil.cs

```diff
@@ -205,9 +205,7 @@ private static XmlElement GetXmlElement(string targetZone, FrameworkName fn)
                         {
                             try
                             {
-#pragma warning disable CA2000 // Dispose objects before losing scope is suppressed because it is managed by FileStream above.
-                                var sr = new StreamReader(fs);
-#pragma warning restore CA2000 // Dispose objects before losing scope
+                                using var sr = new StreamReader(fs, Encoding.UTF8, detectEncodingFromByteOrderMarks: true, 1024, leaveOpen: true);
```

Comment:
nit: Can this be just:
```
new StreamReader(fs, leaveOpen: true);
```
?

Reply:
nope (:
<img width="1778" alt="image" src="https://github.com/dotnet/msbuild/assets/95473390/cf88a249-1c25-4e6e-9a22-1087116257f8">

Reply:
That's odd, the constructor declares all parameters except for Stream as optional 😢